core/src/kiso.rs

```diff
@@ -27,7 +27,7 @@ pub struct KisoHandle {
 
 impl KisoHandle {
     /// Spawn a new actor
-    pub fn new(state: Configuration) -> Self {
+    pub fn new(state: Root) -> Self {
```

Comment:
```suggestion
    pub fn new(state: IrohaConfig) -> Self {
```

we have no idea what `Root` is